src/Standards/PSR2/Tests/ControlStructures/ControlStructureSpacingUnitTest.inc

```diff
@@ -45,3 +45,9 @@ foreach (  $something as $blah => $that  ) {}
 // @codingStandardsChangeSetting PSR2.ControlStructures.ControlStructureSpacing requiredSpacesBeforeClose 0
 
 $binary = b"binary string";
+
+declare(ticks=1) {} // OK
```

Comment:
I used `declare(ticks=1)` instead of `declare(strict_types=1)`, because I wasn't sure if it is OK for the test file to be _invalid_.

Reply:
It's ok for a test file to be invalid. It's tokenized after all, not parsed.